Handle OCI Imported Components When Creating a Differential Package (#1714)
defenseunicorns/zarf: docs/3-create-a-zarf-package/1-zarf-packages.md

The process of defining and creating a package is also elaborated on in detail in the [Creating a Zarf Package Tutorial](../5-zarf-tutorials/0-creating-a-zarf-package.md).

### Creating Differential Packages

> If you have already created a Zarf package and want to create a new package to capture updates that you have made to your package you would normally have to re-create the entire package from scratch, including all of the parts of the package that might not have changed at all. Depending on your workflow, you may only want to create a package that contains the artifacts that have changed since the last time you built your package. This can be achieved by using the `--differential` flag while running the `zarf package create` command. You can use this flag to point to an already built package you have locally or to a package that has been previously [published](../5-zarf-tutorials/7-publish-and-deploy.md#publish-package).

Reviewer: ```suggestion
If you already have a Zarf package and you want to create an updated package you would normally have to re-create the entire package from scratch, including things that might not have changed. Depending on your workflow, you may  want to create a package that only contains the artifacts that have changed since the last time you built your package. This can be achieved by using the `--differential` flag while running the `zarf package create` command. You can use this flag to point to an already built package you have locally or to a package that has been previously [published](../5-zarf-tutorials/7-publish-and-deploy.md#publish-package) to a registry.
```
package package package package package lol
